src/iris/api.py

```diff
@@ -1988,8 +2032,13 @@ def on_get(self, req, resp):
         if where:
             query = query + ' WHERE ' + ' AND '.join(where)
 
+        if order_by is not None:
+            query += ' ORDER BY %s %s' % (incident_order_by_fields[order_by], order)
+        else:
+            query += ' ORDER BY `incident`.`created` DESC, `incident`.`id` DESC'
```

Comment:
It looks like this always adds an ORDER BY clause even if one isn't requested. Depending on whether we have in index on the created field, this might be slower than expected. Would it make more sense to only order the result if it is explicitly requested?